sky/clouds/ssh.py

```diff
@@ -252,7 +252,7 @@ def check_single_context(cls, context: str) -> Tuple[bool, str]:
     @classmethod
     def expand_infras(cls) -> List[str]:
         return [
-            f'{cls.canonical_name()}/{c.lstrip("ssh-")}'
+            f'{cls.canonical_name()}/{c.lstrip("ssh-")}' if c.startswith("ssh-") else "."
```

Comment:
why not just not use lstrip? also, why are we using `.` when it is not starting with ssh-?